addons/hr_recruitment_survey/views/survey_survey_views.xml

```diff
@@ -0,0 +1,88 @@
+        <field name="model">survey.survey</field>
+        <field name="inherit_id" ref="survey.survey_survey_view_form"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='survey_type']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <button name="action_send_survey" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </button>
+            <button name="action_start_session" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </button>
+            <button name="action_archive" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </button>
+            <xpath expr="//label[@for='is_time_limited']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//div[@name='is_time_limited']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//field[@name='scoring_type']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//label[@for='certification']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//div[@name='certification']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//group[@name='live_session']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="survey_survey_view_kanban" model="ir.ui.view">
+        <field name="name">survey.survey.view.kanban.inherit.recruitment</field>
+        <field name="model">survey.survey</field>
+        <field name="inherit_id" ref="survey.survey_survey_view_kanban"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <button name="action_send_survey" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </button>
+            <button name="action_start_session" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </button>
+        </field>
+    </record>
+
+    <record id="survey_survey_action_recruitment" model="ir.actions.act_window">
+        <field name="name">Interviews</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">survey.survey</field>
+        <field name="domain">[('survey_type', '=', 'recruitment')]</field>
+        <field name="view_ids" eval="[(5, 0, 0),
```

Comment:
I didn't say it will be rolled back.

And the why is because if you want something, sometimes you have to fight for it.
What do you think is clearer, `(5, 0, 0)`or `Command.clear()` (your guess)?

Reply:
``Command`` is deprecated, use numbers. So keep the 5, which is a nice number.

Reply:
![8iswwy](https://github.com/odoo/odoo/assets/90857153/e76482c8-6528-4b9f-a964-b86e961e6c5e)

Reply:
Haha, nice try. However it is time to stop pushing for deprecated stuff @flch-odoo . Sometimes we have to acknowledge the world is moving forward and we have to go with it. Numbers it will be !